examples/distributed_inference/tensor_parallel_llama3.py

```diff
@@ -51,7 +51,7 @@
             "use_python_runtime": True,
             "workspace_size": 1 << 33,
             "debug": False,
-            "use_aot_joint_export": False,
+            "use_distributed_mode_trace": True,
```

Comment:
would llama3 be supported now after the removal of complex tensor handling ? If so, is this file required?